pkgs/by-name/um/umockdev/package.nix

```diff
@@ -65,11 +74,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   buildInputs = [
     glib
-    systemdMinimal
+    udev'
```

Comment:
`buildInputs` should probably just get `udev` regardless of whether it's the systemd implementation or not. It only needs `systemdMinimal` for the check phase, and only for `udevadm`, so `systemdMinimal` shouldn't be in `buildInputs`. Plain `udev` should be.